core/src/main/java/com/graphhopper/routing/util/CarFlagEncoder.java

```diff
@@ -158,6 +158,9 @@ public int defineWayBits(int index, int shift) {
 
     protected double getSpeed(ReaderWay way) {
         String highwayValue = way.getTag("highway");
+        if (way.hasTag("motorroad", "yes")) {
```

Comment:
motorroad=yes tag MUST be used with one of the highway values, which is mostly trunk or primary. Maybe I should ignore this tag when highway=motorway because it is invalid. (it is intended for non-motorways only)